packages/obojobo-document-engine/server/attempt-end.js

```diff
@@ -334,119 +337,127 @@ const insertAttemptScoredEvents = (
 		})
 }
 
-const reloadAttemptStateIfReviewing = (
-	attemptId,
-	draftId,
-	attempt,
-	draftDocument,
-	user,
-	isPreview,
-	attemptHistory
-) => {
-	const assessmentNode = attempt.assessmentModel
-
-	// Do not reload the state if reviews are never allowed
-	if (assessmentNode.node.content.review === 'never') {
-		return null
-	}
-
-	const isLastAttempt = attempt.number === assessmentNode.node.content.attempts
-
-	// Do not reload the state if reviews are only allowed after the last
-	// attempt and this is not the last attempt
-	if (assessmentNode.node.content.review === 'no-attempts-remaining' && !isLastAttempt) {
-		return null
-	}
-
-	const assessmentProperties = loadAssessmentProperties(
-		draftDocument,
-		attempt,
-		user,
-		isPreview,
-		attemptHistory
-	)
-
-	const state = attemptStart.getState(assessmentProperties)
-	// Not ideal, but attempt-start needs this as a recursive structure to send
-	// client promises
-	state.questions = state.questions.map(q => q.toObject())
-
-	// If reviews are always allowed, reload the state for this attempt
-	// Each attempt's state will be reloaded as it finishes
-	if (assessmentNode.node.content.review === 'always') {
-		return Assessment.updateAttemptState(attemptId, state)
-	}
-
-	// If reviews are allowed after last attempt and this is the last attempt,
-	// reload the states for all attempts
-	if (assessmentNode.node.content.review === 'no-attempts-remaining' && isLastAttempt) {
-		// Reload state for all previous attempts
-		return Assessment.getAttempts(
-			assessmentProperties.user.id,
-			draftId,
-			isPreview,
-			assessmentProperties.id
-		).then(result => {
-			result.attempts.map(attempt => {
-				attempt.state.qb = recreateChosenQuestionTree(
-					attempt.state.qb,
-					assessmentProperties.draftTree
-				)
-
-				const newQuestions = []
-
-				attempt.state.questions.forEach(question => {
-					newQuestions.push(getNodeQuestion(question.id, assessmentProperties.draftTree))
-				})
-
-				attempt.state.questions = newQuestions
-
-				return Assessment.updateAttemptState(attempt.attemptId, attempt.state)
-			})
-		})
-	}
-
-	logger.error(`Error: Reached exceptional state while reloading state for ${attemptId}`)
-	return null
-}
-
-const recreateChosenQuestionTree = (node, assessmentNode) => {
-	if (node.type === QUESTION_NODE_TYPE) {
-		return getNodeQuestion(node.id, assessmentNode)
-	}
-
-	const newChildren = []
+// const reloadAttemptStateIfReviewing = (
```

Comment:
This is no longer used, but I want to keep it in the PR in case I unknowingly deleted important functionality. Will remove be pull.